apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -49,6 +40,21 @@ defmodule Lexical.RemoteControl do
     end
   end
 
+  def wait_until_connected(node, timeout \\ 5_000)
```

Comment:
I know using `wait_until_connected/2` is a little bit strange here, but I have no idea without it.

```elixir
    receive do
      {:nodeup, ^node, _} ->
        :ok
    after
```
    
I tried monitor_nodes, and tried to receive the `nodeup` message, then `Node.connect(node)`, but it won't work, because the `nodeup` message only appears when `true = Node.connect`